fix 'make install-toolkit' by copying header files in 'common' separately
patch created by Kenneth Hoste (HPC-UGent), based on patch by Kurt Lust (UAntwerpen)
--- ncbi-blast-2.6.0+-src/c++/src/build-system/Makefile.in.top.orig	2017-05-22 13:35:40.236430282 +0200
+++ ncbi-blast-2.6.0+-src/c++/src/build-system/Makefile.in.top	2017-05-22 13:38:37.179080048 +0200
@@ -51,7 +51,8 @@
 	    done
 	cd $(includedir0) && find * -name CVS -prune -o -print |\
             cpio -pd $(pincludedir)
-	$(INSTALL) -m 644 $(incdir)/* $(pincludedir)
+	$(INSTALL) -m 644 $(incdir)/*.h $(pincludedir)
+	$(INSTALL) -m 644 $(incdir)/common/*.h $(pincludedir)/common
 ## set up appropriate build and status directories somewhere under $(libdir)?
 
 install-gbench:
